google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2068,6 +2068,13 @@ Blob create(
    */
   WriteChannel writer(BlobInfo blobInfo, BlobWriteOption... options);
 
+  /**
+   * accept signURL and return a channel for writing content.
```

Comment:
Can you make this more consistent with the other documentation in this class? i.e. present tense ("Accepts" instead of "accept") and provide an example?

Reply:
@JesseLovelace changes done with example